packages/react-native/README.md

```diff
@@ -50,7 +50,19 @@ Error reports can be [customized](https://docs.honeybadger.io/lib/javascript/gui
 Some native errors on Android may not be recorded if they cause an immediate crash of the app before the notice makes it to Honeybadger. 
 
 ### Source Maps
-The procedure to generate and upload sourcemaps for react-native is currently [under development](https://github.com/honeybadger-io/honeybadger-js/issues/1029).
+To generate and upload source maps to Honeybadger, use the following command:
+```shell
+npx honeybadger-upload-sourcemaps --apiKey <your project API key> --revision <build revision>
```

Comment:
Can we please mention the `--skip-upload` flag here as well?

Reply:
Yep. Done!